force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,16 +84,19 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
-
+        List<String> myWords = sentence.split(' ');
         // Create a set to store the unique words
-
+        Set<String> wordsSet = new Set<String>();
```

Comment:
You can build a Set directly from a List, which makes it unique. No need for for loop.